AutomatedTesting/Gem/PythonTests/CMakeLists.txt

```diff
@@ -56,6 +56,9 @@ add_subdirectory(streaming)
 ## Smoke ##
 add_subdirectory(smoke)
 
+## Terrain ##
+add_subdirectory(terrain)
```

Comment:
Please use "Terrain" instead of "terrain". It's a bit of a mess right now but we decided to go with UpperCamelCase

Reply:
Done.